### PR TITLE
fixed errors

### PR DESCRIPTION
**Pull request by minhochoi1983**

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+    right: 100px;
+    top: 100px;
+    color: white;
+    display: block;
+    vertical-align: middle;
+    
+}
+
+.cart-img {
+    width: 50px;
```
Comment (MadEste) [inline style.css]: Be careful about using px in the next assignment. px are not responsive units and should not be used ANYWHERE in your css except to perhaps set the line width of an element.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,455 @@
...
+    height: 500px;
+    overflow: hidden;
+    
+}
+
+
+
+.header-left {
+    position: absolute;
+    top: 100px;
```
Comment (MadEste) [inline style.css]: Do not use px for positioning moving forward.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+    font-weight: 900;
+    color: #164668;
+}
+
+nav ul li .menu-lists {
+    color: #164668;  
+}
+nav {
+    background-color: #EBEBEB;
+    height: 50px;
```
Comment (MadEste) [inline style.css]: try em, rem, or %

**Comment by minhochoi1983** [comment]
Edited all px to rem

**Comment by minhochoi1983** [comment]
added lesson 4 parts

**Pull request closed without merging**